addons/fullscreen/addon.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "Enhanced fullscreen",
+  "description": "Adds several tweaks and customization options to the project player's fullscreen mode, such as syncing with your browser's fullscreen mode (F11), darkening the background behind the project player, and hiding the green flag toolbar.",
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://github.com/lisa-wolfgang"
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "If you hide the toolbar and don't have the fullscreen sync option enabled, you'll need to use the Esc key on your keyboard to exit the project player's fullscreen mode.",
+      "id": "hideToolbarNotice"
+    },
+    {
+      "type": "notice",
+      "text": "If you use the F11 key to enter fullscreen, the Esc key will not fully exit browser fullscreen. If you enable the editor fullscreen option and you're in the editor, when you use F11 and then click the stage button, neither the Esc key nor the stage button will exit browser fullscreen.",
```

Comment:
Is there no JavaScript fix for this? This notice is so weird and hard to understand. Is it actually impossible to make this work as we'd intend?

Reply:
This is unfortunately the case. Browser-level full screen cannot be controlled by JS, so when the user enters browser full screen with F11 and then tries to use the Esc key to exit full screen, it is impossible to also toggle browser full screen.

The best solution is probably to disable the F11 key behavior and rely solely on Esc and the full screen button. A plus with this is that the editor setting is rendered completely unnecessary because the user has complete control over their browser full screen by default. This doesn't really fix the problem entirely, as the user can still use F11 and then click the full screen button, but at least the user won't expect browser full screen to be toggled automatically for them in that case.

Reply:
Never mind, `e.preventDefault()` can be used, and we can then toggle document-level full screen ourselves. However, I still think that keeping `F11` out of the picture makes the addon easier to understand.